### DISCUSSION TITLE
Is it possible to properly train on MacBook?

### DISCUSSION THREAD

**Discussion by nagyniki017** [voicepaw/so-vits-svc-fork#1046] [Q&A]
I have a MacBook M3 Pro which I wanted to use to train.

The problem I am facing is that when I run infer on the M3 trained model, the result is some unbearable noise, not even near to human voice.
I was curious what would happen if I continued the training which I have started on Colab. It gets totally messed up.
So it's giving really bad results when I start the training from scratch or when I try to continue a previous training.

Is there some setup or config I am missing to be able to get it working?

**Comment by Sarang0218** [comment]
I have the same problem! The results are still noisy after 30k steps! (starting from scratch)
